core/arch/arm/plat-imx/link.mk

```diff
@@ -1,3 +1,15 @@
 include core/arch/arm/kernel/link.mk
 
 all: $(link-out-dir)/tee-raw.bin
+
+cleanfiles += $(link-out-dir)/entry_point_address.txt
+$(link-out-dir)/entry_point_address.txt: $(link-out-dir)/tee.elf
+	@$(cmd-echo-silent) '  GEN     $@'
+	$(q)$(READELFcore) -h $(link-out-dir)/tee.elf | grep "Entry point address" | awk '{print $$4}' > $@
+
+uTee: $(link-out-dir)/uTee
+cleanfiles += $(link-out-dir)/uTee
+$(link-out-dir)/uTee: $(link-out-dir)/tee-raw.bin $(link-out-dir)/entry_point_address.txt
+	@$(cmd-echo-silent) '  MKIMAGE $@'
+	$(q)addr=`cat $(link-out-dir)/entry_point_address.txt`; \
```

Comment:
Wouldn't this be almost the same as `CFG_TZDRAM_START`?

Reply:
Yes indeed

Reply:
So why are you taking the detour via `entry_point_address.txt`?
Not that it matters much, but it seems a bit more complicated.

Reply:
For imx6 platforms, `CFG_TZDRAM_START` is equal to `(0x80000000 - 0x01e00000 - 0x00200000 + 0x20000000)`. So I cannot do the following as it fails: `mkimage -A arm -O linux -C none -a $(CFG_TZDRAM_START) -e $(CFG_TZDRAM_START) -d $< $@`.
Do you have any idea on how to use `CFG_TZDRAM_START` here ?

Reply:
I see, let's keep it as it is then.

Reply:
I guess that's what we are looking for! Thanks!